ci/install_travis.sh

```diff
@@ -148,6 +148,9 @@ if [ -e ${REQ} ]; then
     time bash $REQ || exit 1
 fi
 
+# test commit
+conda install bzip2
```

Comment:
if we need this is should be somewhat selective and should skip tests if it's not installed

Reply:
It's an issue with the recipe in the defaults channel. It's being rebuilt.

I'm kind of surprised that pytables doesn't raise an exception on write here. They fall back to zlib with a warning.

Reply:
yep u to want to raise an issue there